test/Microsoft.AspNetCore.Server.KestrelTests/ChunkedRequestTests.cs

```diff
@@ -107,12 +109,16 @@ public static TheoryData<ServiceContext> ConnectionFilterData
                     await connection.Receive(
                         "HTTP/1.1 200 OK",
                         "Connection: keep-alive",
+                        "Date: " + testContext.DateHeaderValue,
```

Comment:
Do you think this is an OK solution to assert on the `Date` header? I see `connection.ReceiveStartsWith("Date:")` is used a few places. Would you rather I used that instead?